dracut-initramfs-restore.sh.in

```diff
@@ -39,8 +39,8 @@ elif [[ -d /boot/efi/loader/entries || -L /boot/efi/loader/entries ]] \
     IMG="/boot/efi/$MACHINE_ID/$KERNEL_VERSION/initrd"
 elif [[ -f /lib/modules/${KERNEL_VERSION}/initrd ]]; then
     IMG="/lib/modules/${KERNEL_VERSION}/initrd"
-elif [[ -f /boot/initramfs-${KERNEL_VERSION}.img ]]; then
-    IMG="/boot/initramfs-${KERNEL_VERSION}.img"
+elif [[ -f /boot/@INITRDPREFIX@-${KERNEL_VERSION}@INITRDSUFFIX@ ]]; then
+    IMG="/boot/@INITRDPREFIX@-${KERNEL_VERSION}@INITRDSUFFIX@"
```

Comment:
For full flexibility: What do you think about moving `-` also to the prefix (i.e. `initrdprefix=initramfs-`)?

Reply:
sure. applied

Reply:
I would actually prefer _not_ doing that. Is there a compelling reason to make the separators inconsistent?

Reply:
What does "inconsistent" mean here. If you want to have dracut to make `6.1.0-12-amd64.initrd` then that's valid and not possible with a partially-fixed prefix.